src/components/Footer.tsx

```diff
@@ -0,0 +1,66 @@
+import { ConstantData, NAVIGATION_DATA, SOCIAL_DATA } from '@constants';
+import {
+  FacebookIcon, LinkedInIcon, TwitterIcon, YoutubeIcon,
+} from '@icons';
+import Image from 'next/image';
+import Link from 'next/link';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { BrlLogo } from '../assets/images';
+
+interface IProps { }
+
+export const Footer: React.FC<IProps> = () => {
+  const { t } = useTranslation();
+  const FIRST_THREE = NAVIGATION_DATA.slice(0, 3);
+  const LAST_THREE = NAVIGATION_DATA.slice(3);
+
+  const footerLinks = (links: ConstantData[]) => (
+    <ul>
+      {links.map((item) => (
+        <li key={item.value}>
+          <Link href={item.value}>
+            <a>{item.title}</a>
+          </Link>
+        </li>
+      ))}
+    </ul>
+  );
+
+  return (
+    <footer>
+      <div className="footer-link-section">
+        {footerLinks(FIRST_THREE)}
+        <Link href="/">
+          <a className="footer-logo">
+            <Image src={BrlLogo.src} alt="BRL-CAD" width={72} height={72} />
+          </a>
+        </Link>
+        {footerLinks(LAST_THREE)}
+      </div>
+      <div className="footer-social">
+        {SOCIAL_DATA.map((item) => {
+          const Icon = {
+            facebook: FacebookIcon,
+            linkedin: LinkedInIcon,
+            twitter: TwitterIcon,
+            youtube: YoutubeIcon,
+          }[item.title as 'facebook' | 'linkedin' | 'twitter' | 'youtube'];
+          return (
+            <Link key={item.value} href={item.value}><a><Icon /></a></Link>
+          );
+        })}
+      </div>
+      <Link href="https://bestpractices.coreinfrastructure.org/projects/66">
```

Comment:
You can put the links in a constant file and map them from there.

Reply:
This one is fixed and pushed

Reply:
Can you link the changes here? @divyanshugarg36

Reply:
This is the commit showing these changes
https://github.com/divyanshugarg36/brl-cad-web/pull/3/commits/28b71fd6805a84b08c247b97acf06a0160ee441e#diff-11bdf8babe536adbf336e8de7bdfe65b3a86d4d7011c51da852b69e8d3248af8R24